engine/class_modules/priest/sc_priest.cpp

```diff
@@ -545,11 +543,22 @@ struct smite_t final : public priest_spell_t
     if ( priest().buffs.wrath_unleashed->check() )
     {
       d *= 1.0 + priest().buffs.wrath_unleashed->data().effectN( 1 ).percent();
+      sim->print_debug( "Smite damage modified by {} (new total: {}), from wrath unleashed",
```

Comment:
```suggestion
      sim->print_debug( "Smite damage modified by {} (new total: {}), from wrath_unleashed",
```

I would _highly_ suggest using `_` name format when you put spell names in debug. 99% of lines for a particular thing are formatted like this, so conforming to that makes it easier when searching. So in this case use `wrath_unleashed`